<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />  
    <!--  <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" /> -->
    <rosparam file="$(find my_navigation)/config/mbot/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find my_navigation)/config/mbot/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find my_navigation)/config/mbot/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find my_navigation)/config/mbot/global_costmap_params.yaml" command="load" />

    <rosparam file="$(find my_navigation)/config/mbot/move_base_params.yaml" command="load" />
    <rosparam file="$(find my_navigation)/config/mbot/teb_local_planner_params.yaml"  command="load" /> 
   <!--<rosparam file="$(find my_navigation)/config/mbot/dwa_local_planner_params.yaml" command="load" /> -->
  </node>
  
</launch>
